components/brave_wallet_ui/common/async/base-query-cache.ts

```diff
@@ -137,7 +113,7 @@ export class BaseQueryCache {
   }
 
   clearSelectedAccount = () => {
-    this._selectedAccountAddress = undefined
+    this.clearAccountsRegistry()
```

Comment:
Can we accomplish this without refreshing the accounts list?

Reply:
Not now. Let's address this if it noticeably affects performance.